### PR TITLE
Modify modules folder

### PR DESCRIPTION
**Pull request by Athaliah8**
## In This Repo

- Created a new directory and initialized it with git.
- Created a directory called modules.
- Added index JS, CSS and HTML files.
- Used ES6 syntax.
- Divided code into modules. 
- Kept all functionalities of the app without errors.
- Refactored methods and functions to arrow functions.
- Used let and const in the correct way.
-  Used a library Luxon and imported it as a module.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    storedBooks.map((book) => Layout.bookList(book));
+  }
+
+  static bookList(book) {
+    const collection = document.getElementById('collection');
+    collection.innerHTML += `
+      <li id='${book.id}' class='listItem'>
+        <p>"${book.title}" by ${book.author} </p>
+        <button id='delete' type='button' onclick ='Layout.deleteBook(${book.id})'>Remove</button>
+      </li>
```
Comment (AmaduKamara) [inline index.js]: - [ ] When I click on the Remove button it's expected to delete a book from the list of books. Currently, it does not seem to be working as expected. Adding books is working fine but deleting a book is not working. Kindly fix that feature so your project will be complete for the milestone 👍

**Review by AmaduKamara — APPROVED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @Athaliah8,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly 👍 
✅ Linter checks passed 👍 
✅ Good descriptive PR 👍 
✅ Add book feature works fine as expected 👍 
✅ Used es6 modules and function arrow syntax 👍 

## Required Changes ♻️
_Check the comments under the review._

- [x] Kindly make your commit messages descriptive. For instance, if you work on the delete feature, you could write your commit message as in - modify delete book functionality. 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.js (hunk 1 above)
Comment (AmaduKamara) [reply index.js:52]: Please ignore the review headline above. I meant to request changes and not approve. Please work on the changes as highlighted above.

image

**Comment by AmaduKamara** [comment]
Please ignore the review headline above. I meant to request changes and not approve. Please work on the changes as highlighted above.

image

**Review by jerryowusu — APPROVED** [review]
Hi @Athaliah8 :wave:  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

200w


## Highlights
Project is working fine :+1: 
All linters are passing :+1: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**